### ISSUE TITLE
ctypes.Structure incorrect not callable error 

### ISSUE THREAD

**Issue by nathanpage-credo** [microsoft/pyright#2829]
**Describe the bug**
The latest pylance release introduced a bug with `ctypes.Structure` elements falsely claiming that they are not callable

**To Reproduce**

```python
import ctypes as ct

class Foo(ct.Structure):
    _fields_ = [("chip_id", ct.c_uint8), ("slice_id", ct.c_uint8), ("sys_id", ct.c_uint8)]


a = Foo()  # this will error saying: Object of type "_CField" is not callable
```

**Expected behavior**
This should not error.

**VS Code extension or command-line**

vscode extension: pylance v2022.1.0

Underneath uses pyright 1.1.204

**Comment by erictraut** [comment]
Thanks for the bug report, and apologies for the regression. The `ct.Structure` classes uses a custom metaclass with a `__getattr__` method, which was breaking some of the logic that was introduced in pyright 1.1.199 to handle `__call__` methods on metaclasses.

This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.206, which I just published. It will also be in the next release of pylance.
